lib/sidekiq/worker_killer.rb

```diff
@@ -125,6 +131,10 @@ def sidekiq_process
     end || raise("No sidekiq worker with identity #{identity} found")
   end
 
+  def identity
+    config[:identity] || config["identity"]
+  end unless method_defined?(:identity)
```

Comment:
include Sidekiq::Util  will have defined this. Else we define it and call the interface added by Sidekiq::ServerMiddleware